xee/ext.py

```diff
@@ -1037,7 +1060,14 @@ def open_dataset(
       executor_kwargs (optional): A dictionary of keyword arguments to pass to
         the ThreadPoolExecutor that handles the parallel computation of pixels
         i.e. {'max_workers': 2}.
-
+      tile_fetch_kwargs (optional): The necessary kwargs like `max_retries`,
```

Comment:
Could you update the description to make this as explicit as possible (you'll have to do some line re-wrapping):

```
      tile_fetch_kwargs (optional): Exponential backoff kwargs passed into the xarray function to index the array (`robust_getitem`). Supported values are:
        - `initial_delay`, the initial delay in milliseconds before the first retry. Defaults to 500.
        - `max_retries`, the maximum number of retry attempts. Defaults to 6.
```